dali/pipeline/executor/executor.h

```diff
@@ -134,6 +132,8 @@ class DLL_PUBLIC Executor : public ExecutorBase, public WorkspacePolicy, public
 
   void SetupOutputQueuesForGraph();
 
+  int &QueueDepth(OpType stage);
```

Comment:
(not important) could have put the impl here since it's a one liner and avoid having to add all the template stuff in the implementation